dbt_project.yml

```diff
@@ -1,7 +1,8 @@
 name: 'dbt_utils'
 version: '0.1.0'
 
-require-dbt-version: [">=0.18.0", "<0.20.0"]
+require-dbt-version: [">=0.19.0", "<0.20.0"]
```

Comment:
Can we allow version 0.20.0 since that's already where dbt core master is?

Reply:
0.20.0 is going to require some breaking changes to this package that aren't backwards compatible. See https://github.com/fishtown-analytics/dbt-utils/compare/dev/0.7.0...feature/0.20.0-compatibility.

@clrcrl Up to you if you want to coordinate 0.7.0 release with 0.20.0 release. I think it would make sense to release 0.7.0 ahead of time, so that v0.19 users can benefit from the new utils features, and then upgrade to dbt v0.20 + dbt-utils v0.8 when they can.

Reply:
Gotcha, I will use the `feature/0.20.0-compatibility` branch of `dbt-utils` in my `packages.yml`. Carry on...